src/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.cs

```diff
@@ -198,11 +198,8 @@ public sealed partial class OptionTextAttribute : System.Attribute
     public sealed partial class ProjectData
     {
         internal ProjectData() { }
-        [System.Security.SecuritySafeCriticalAttribute]
```

Comment:
There is no difference between SafeCritical and transparent (ie no annotation) in the contract. 

In other words, changing from SafeCritical to transparent or vice versa, is not a contract breaking change.